source/user-manual/reference/ossec-conf/fluent-forward.rst

```diff
@@ -0,0 +1,192 @@
+.. Copyright (C) 2018 Wazuh, Inc.
+
+.. _reference_ossec_fluent_forward:
+
+fluent-forward
+==============
+
+.. topic:: XML section name
+
+	.. code-block:: xml
+
+		<fluent_forward>
+		</fluent_forward>
+
+This configuration section is used to configure the Fluentd forwarder module.
+
+Options
+-------
+
+- `enabled`_
+- `socket_path`_
+- `tag`_
+- `address`_
+- `port`_
+- `shared_key`_
+- `ca_file`_
+- `user`_
+- `password`_
+- `timeout`_
+
+enabled
+^^^^^^^
+
+This indicates if the module is enabled or disabled.
+
++--------------------+--------------+
+| **Default value**  | n/a          |
++--------------------+--------------+
+| **Allowed values** | yes, no      |
++--------------------+--------------+
+
+socket_path
+^^^^^^^^^^^
+
+This indicates the path of the UPD socket to be read from. The socket will be created at runtime in the designated path.
```

Comment:
Typo: ~UPD~ UDP.
Also, use "listen to" instead of "read from". I think that makes this sentence clearer because it's related to sockets.